user-guide/Advanced_Functionality/Swarming/SwarmingFAQ.md

```diff
@@ -0,0 +1,72 @@
+---
+uid: SwarmingFAQ
+---
+
+# Frequently Asked Questions
+
+## Can I Swarm X/Y/Z?
+
+Swarming will gradually become available for more and more functionality in DataMiner.
+As of DataMiner 10.5.1, only basic elements are supported.
+
+## Does the ID of an element change when it is Swarmed?
+
+No, the identifier of the element stays the same.
```

Comment:
Maybe an example of an identifier "123/456" and where you could see or have to use that reference

Or to the question on DataMinerID below

Reply:
redirected to the next question instead